include/solvers/vof_linear_subequations_solver.h

```diff
@@ -0,0 +1,242 @@
+// SPDX-FileCopyrightText: Copyright (c) 2021-2024 The Lethe Authors
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception OR LGPL-2.1-or-later
+
+#ifndef lethe_vof_linear_subequations_solver_h
+#define lethe_vof_linear_subequations_solver_h
```

Comment:
I would keep this file more general, in case we end up a non linear system at some point.

```suggestion
#define lethe_vof_subequations_solver_h
```

Reply:
For non linear solvers, I think we should opt for a similar architecture to other non-linear equations, i.e. going through the a similar mecanism as the PhysicsSolver.

Here, we had an exception were equations are solved directly in their linear form.